zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -0,0 +1,996 @@
+        self._lvm.create_lv_from_absolute_path(
+            metadata_path,
+            INITIAL_LV_SIZE,
+            tag=LV_METADATA_TAG,
+            lock=False,
+            exact_size=True,
+        )
+        initialize_metadata_lv(metadata_path, INITIAL_LV_SIZE)
+        logger.info("created and initialized metadata LV %s", metadata_path)
+
+    def _lv_list_func(self, vg):
+        r, o = self._bash.bash_ro(
```

Ruff (0.15.6)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)